src/GeneratedPlaylists/GeneratedPlaylist.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+function GeneratedPlaylist() {
+  return (
+    <section className='generated-playlist'>
+    </section>
+  )
+}
```

Comment:
Wild how it appears everything we've been doing thus far looks exactly like standard JS, despite what we'll get into later on.